crates/contract/tests/inprocess/expired_attestation.rs

```diff
@@ -322,6 +322,19 @@ fn test_clean_tee_status_removes_non_participants() {
     assert_eq!(final_participant_count, PARTICIPANT_COUNT);
 }
 
+macro_rules! assert_allowed_code_hashes {
+    ($test_setup:expr, $blocktime_ns:expr, $expected_value:expr $(,)?) => {{
+        set_system_time($blocktime_ns);
+        let res: Vec<[u8; 32]> = $test_setup
+            .contract
+            .allowed_code_hashes()
+            .iter()
+            .map(|hash| *hash.clone())
+            .collect();
+        assert_eq!(res, $expected_value);
+    }};
+}
+
```

Comment:
What's the point of making this a macro as opposed to just having it be a normal function?

```suggestion
fn assert_allowed_code_hashes(
    test_setup: &TestSetup,
    blocktime_ns: u64,
    expected_value: &[[u8; 32]],
) {
    set_system_time(blocktime_ns);
    let res: Vec<[u8; 32]> = test_setup
        .contract
        .allowed_code_hashes()
        .iter()
        .map(|hash| *hash.clone())
        .collect();
    assert_eq!(res, expected_value);
}
```

Reply:
Hmm interesting point. Fair enough :+1: